operator/controllers/seldondeployment_controller.go

```diff
@@ -1072,6 +1072,18 @@ func (r *SeldonDeploymentReconciler) createIstioServices(components *components,
 		}
 	}
 
+	if ready {
```

Comment:
Would this mean that for Ambassador we wouldnt' have the same degree of "precision" on when the service is ready given we don't look at the ambassador CRs?